src/main/java/XmlParser.java

```diff
@@ -1,4 +1,41 @@
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+import org.xml.sax.SAXException;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import java.io.IOException;
+
 public class XmlParser {
+    // proszę o sprawdzenie
     public static void main(String[] args) {
+        String file = args[0];
+        parser(file);
+    }
+
+    public static String parser(String file) {
```

Comment:
Please use OOP principles, not procedural paradigm
https://medium.com/@LiliOuakninFelsen/functional-vs-object-oriented-vs-procedural-programming-a3d4585557f3

Reply:
Also I encourage to read https://www.jakowicz.com/what-is-apie/ in regards to the above